### ISSUE TITLE
Radar with lag and delay

### ISSUE THREAD

**Issue by chaz1to** [drweissbrot/cs-hud#161]
I'm experiencing delay and lag in the radar whenever the focused player enters smoke or gets close to multiple smoke grenades. This happens very often on t-upper/lower on Ancient. I don't know if it's just me, but I've tested it multiple times and it happens consistently.

specs ryzen 5 5600, rtx 2060, 16gb ram

**Comment by drweissbrot** [comment]
could you try limiting the game's FPS (e.g. by using `fps_max 60` in the in-game console) and see if it's still an issue then?

(not a real solution obviously, just a workaround)

**Comment by chaz1to** [comment]
the fps was always limited to 60, i dont tried it without the fps limited

**Comment by drweissbrot** [comment]
Could you share a recording of this?

**Comment by chaz1to** [comment]
of course, https://clips.twitch.tv/LaconicCoweringSangMingLee-cfnz0jl6uZDihMog

**Comment by maxkrizik** [comment]
+ the same

**Comment by 42Firehawk** [comment]
While it probably isn't the issue, there is an issue that LHM has spotted where if there are too many gamestate integration files in the config folder, though with CS2 that has gotten less and less common.

**Comment by KYXT** [comment]
`fps_max 60` helped me, thanks

**Comment by nsnsay** [comment]
@drweissbrot Using absolute positioning without using transform can lead to rendering that does not utilize GPU computation, which can result in significant performance overhead.
